index.html

```diff
@@ -69,33 +67,85 @@
     SHOULD join the mailing lists below and take part in the discussions.</p>
   </section>
   <section>
+      <h2>
+        Dependencies
+      </h2>
+      <p>
+        The terms <dfn><a href=
```

Comment:
Change this one to:
```HTML

<code><dfn><a href="https://html.spec.whatwg.org/multipage/semantics.html#the-link-element">link</a></dfn></code>
```

Then ReSpec will know automatically add `code` around every `<a>link</a>` ref.